cli/commands/root.go

```diff
@@ -23,6 +23,11 @@ const hasuraASCIIText = `
 
 `
 
+var (
+	//IsUpdateCommandEnabled is an ldflag used to disable the automatic updates
+	IsUpdateCommandEnabled string
```

Comment:
Can we have the default value as true so that, the actual build command in the `Makefile` remains unchanged ?